src/requirements/requirement-frontend-utils.ts

```diff
@@ -168,20 +177,22 @@ export function computeFulfillmentCoursesAndStatistics(
   const subRequirementProgress: number[] = eligibleCourses.map(() => 0);
   coursesTaken.forEach(courseTaken => {
     const { courseId } = courseTaken;
-    for (
-      let subRequirementIndex = 0;
-      subRequirementIndex < eligibleCourses.length;
-      subRequirementIndex += 1
-    ) {
-      if (
-        eligibleCourses[subRequirementIndex].includes(courseId) &&
-        subRequirementProgress[subRequirementIndex] < perSlotMinCount[subRequirementIndex]
+    if (!(requirement.disallowTransferCredit && courseIsAPIB(courseTaken))) {
```

Comment:
I'm not sure why the formatting on Github is weird, but the only thing I added was this line. It ensures that a course can only fulfill the requirement if the requirement doesn't have the `disallowTransferCredit` flag (or, if it does, that the course isn't an AP/IB equivalent course).